src/file.ts

```diff
@@ -2179,6 +2182,8 @@ class File extends ServiceObject<File> {
    * @param {object} config Configuration object.
    * @param {string} config.action "read" (HTTP: GET), "write" (HTTP: PUT), or
    *     "delete" (HTTP: DELETE), "resumable" (HTTP: POST).
+   * @param {string} [config.version] The signing version to use. Currently support 'v2' and 'v4' signing.
```

Comment:
For noting the default version, you can do:

```js
@param {string} [config.version='v2']
```

Also, I believe we're sticking to 80 char line length limits.

Here's a small rewrite:

```js
     * @param {string} [config.version='v2'] The signing version to use, either
     *     'v2' or 'v4'.
```

Reply:
Thanks Stephen